bigquery_etl/dryrun.py

```diff
@@ -240,10 +240,14 @@ def dry_run_result(self):
 
     def get_referenced_tables(self):
         """Return referenced tables by dry running the SQL file."""
-        if self.sqlfile not in SKIP and not self.is_valid():
+        # strip file path to 'sql/project/dataset/table/*.sql' format
+        # to be checked against the SKIP list
+        filepath = self.sqlfile[self.sqlfile.find("/sql/") + 1 :]
```

Comment:
Clever. I was assuming we'd need to split on `/`, but this is readable and looks like it should be effective.